### PR TITLE
added meta_marathon_host label

### PR DESCRIPTION
**Pull request by Smithx10**
This pull request should address the criticism received in a prior pull request that hard coded logic in the marathon_sd configuration.  

It addresses the following issue:
https://github.com/prometheus/prometheus/issues/5067

### CODE REVIEW

**Comment by simonpasquier** [comment]
You'd need to update https://github.com/prometheus/prometheus/blob/master/docs/configuration/configuration.md too. And sign your commit for DCO.

File: discovery/marathon/marathon.go [hunk 1]
```diff
@@ -49,7 +49,8 @@ const (
 	portIndexLabel model.LabelName = metaLabelPrefix + "port_index"
 	// taskLabel contains the mesos task name of the app instance.
 	taskLabel model.LabelName = metaLabelPrefix + "task"
-
+	// hostLabel contains the mesos task host address of the app instance
```
Comment (brian-brazil) [inline discovery/marathon/marathon.go]: full stop

Comment (Smithx10) [reply discovery/marathon/marathon.go]: I believe the full stop was used to group port labels together. 

So the stop probably should be after the hostLabel Line correct?

See the following:
https://github.com/prometheus/prometheus/blob/master/discovery/marathon/marathon.go#L52

Comment (brian-brazil) [reply discovery/marathon/marathon.go]: The style in Go is to have comments as full sentences, which includes a full stop. So all of these should have a full stop.

File: docs/configuration/configuration.md [hunk 2]
```diff
@@ -832,6 +832,7 @@ The following meta labels are available on targets during [relabeling](#relabel_
 * `__meta_marathon_app`: the name of the app (with slashes replaced by dashes)
 * `__meta_marathon_image`: the name of the Docker image used (if available)
 * `__meta_marathon_task`: the ID of the Mesos task
+* `__meta_marathon_host`: the Host Address of the Mesos task
```
Comment (brian-brazil) [inline docs/configuration/configuration.md]: host address, neither are proper nouns

**Comment by roidelapluie** [comment]
@Smithx10 Are you still interested by this?

**Comment by Smithx10** [comment]
@roidelapluie,

Nope.  Luckily I no longer have to use DC/OS.  :)

**Comment by codesome** [comment]
This is still relevant. @Smithx10 could you fix the review comments and the DCO? Thanks!

**Comment by bwplotka** [comment]
Sounds like this is relevant, but author is not interested to pursue this. Also Mesos is on path to deprecation. 

If anyone wants to use it and fix it, this closed PR will be a good inspiration (: 

Thanks @Smithx10 anyway!

**Pull request closed without merging**